File: javascript/coordenadas.js
const x = parseFloat(prompt("Valor de X"));
const y = parseFloat(prompt("Valor de Y"));

if (x > 0 && y > 0) {
    console.log("Q1");
} else if (x < 0 && y > 0) {
    console.log("Q2");
} else if (x < 0 && y < 0) {
    console.log("Q3");
} else if (x > 0 && y < 0) {
    console.log("Q4");
} else if (x == 0 && y != 0) {
    console.log("Eixo Y");
} else if (x != 0 && y == 0) {
    console.log("Eixo X");
} else if (x == 0 && y == 0) {
    console.log("Origem");
}

File: python/coordenadas.py
x = float(input("Valor de X: "))
y = float(input("Valor de Y: "))

if y > 0 and x > 0:
    print("Q1")
elif y > 0 and x < 0:
    print("Q2")
elif y < 0 and x < 0:
    print("Q3")
elif y < 0 and x > 0:
    print("Q4")
elif x == 0 and y != 0:
    print("Eixo Y")
elif x != 0 and y == 0:
    print("Eixo X")
else:
    print("Origem")
